cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -316,10 +316,24 @@ type LocalEtcd struct {
 
 // ExternalEtcd describes an external etcd cluster
 type ExternalEtcd struct {
-
-	// Endpoints of etcd members. Useful for using external etcd.
+	// Endpoints of etcd members. Required when using external etcd.
+	// Specifies the client URLs (usually gRPC endpoints) for etcd communication.
+	// By default, these endpoints handle both gRPC traffic (primary etcd protocol)
+	// and HTTP traffic (metrics, health checks). However, if HTTPEndpoints is configured,
+	// the gRPC and HTTP traffic can be separated for better security and performance.
+	// Corresponds to etcd's --listen-client-urls configuration.
 	// If not provided, kubeadm will run etcd in a static pod.
```

Comment:
same here, we should either remove the sentence or move it in the comment above the type.